R/importCNA.GISTIC2.R

```diff
@@ -0,0 +1,96 @@
+# importCNA.GISTIC2.R
+#
+#
+#'@function Imports copy number aberration data from Firehose database,
+# produces an rCNA object saved as a RDS file.
+#
+#
+#'@param fFHG Local fully qualified path of Firehose GISTIC2 CNA file.
+#'@param dCNA The local fully qualified path of a directory to store the rCNA RDS file(s).
+#'@param silent Boolean. Default: FALSE. Whether or not to write progress information to console.
+#'@param noLog Boolean. Default: FALSE. Whether or not to log results.
+#
+#
+#'@family importM.COSMIC, importM.TCGA, importM.GISTIC2, importM.ProjectGenie
+#
+#'@example
+# fFHG <- "inst/extdata/devCNA.txt"
+# dCNA <- "inst/extdata/dCNA"
+# silent <- FALSE
+# writeLog <- TRUE
+#
+# importCNA.GISTIC2(fFHG, dCNA, silent = FALSE, writeLog = TRUE)
+#
```

Comment:
don't write logFile data in examples. Also, don't need to set arguments to their default state (silent).